frontend/e2e/_gcweb-app.personal-information.home-address.confirm.spec.ts

```diff
@@ -0,0 +1,18 @@
+import AxeBuilder from '@axe-core/playwright';
+import { expect, test } from '@playwright/test';
+
+// Todo: beef up
```

Comment:
**Nitpick:* use `TODO` because this is typically what an IDE looks for.

ex:
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48123208/3916784d-eeb7-4eaf-9f8b-df8b3a4e99b5)